integration/mediation-tests/tests-patches/src/test/resources/artifacts/ESB/passthru/transport/preserveResponseHeaders/passthru-http.properties

```diff
@@ -0,0 +1,42 @@
+#
+#  Copyright (c) 2022, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+## This file contains the configuration parameters used by the Pass-through HTTP transport
+
+## Pass-through HTTP transport specific tuning parameters
+http.socket.timeout=180000
+
+worker_pool_size_core=400
+worker_pool_size_max=500
+#worker_thread_keepalive_sec=60
+#worker_pool_queue_length=-1
+#io_threads_per_reactor=2
+io_buffer_size=16384
```

Comment:
do we need this file, isn't this configured via toml ?

Reply:
The `passthru-http.properties.j2` file was not updated with this `http.response.headers.preserve` property. I have created an issue to track it https://github.com/wso2/api-manager/issues/34